packages/landing/src/components/map.tsx

```diff
@@ -91,7 +91,7 @@ export class Basemaps extends Component<unknown, { isLayerSwitcherEnabled: boole
   };
 
   updateVisibleLayers = (newLayers: string): void => {
-    if (!Config.map.visibleLayers) Config.map.visibleLayers = newLayers;
+    if (Config.map.visibleLayers == null) Config.map.visibleLayers = newLayers;
```

Comment:
I think `visibleLayers` is defined as `string` and not initialised. Might be better to defined as `string|null`. Then you can do this check for initialising here.